WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -186,6 +186,13 @@ private void StepOutputRegistrationPhase()
 							coinjoin = coinjoin.AddOutput(new TxOut(diffMoney, Config.BlameScript));
 							round.LogInfo("Filled up the outputs to build a reasonable transaction because some alice failed to provide its output.");
 						}
+						else
```

Comment:
I think the safest would be to make the fake CoinJoin (blame round) as real as possible - thus adding the coord fee as well to the tx. 
In this way, the same assumptions can be made related to DoS attacks. For example what if a client tricks the coord in a way that it is not sending a small amount output (1 sat greater than `AllowedOutputAmounts.Min`), and then later signs the transaction - thus not paying coord fees at all.

Reply:
> coordination fees included

They aren't.